packages/cmf/scripts/cmf-settings.utils.test.js

```diff
@@ -0,0 +1,9 @@
+const { sortObject } = require('./cmf-settings.utils');
+
+describe('cmf-settings.utils', () => {
+	describe('#sortObject', () => {
+		it('should sort the object', () => {
+			expect(sortObject({ c: 1, a: 2, b: 3 })).toEqual({ a: 2, b: 3, c: 1 });
```

Comment:
toEqual doesn't test the order. You should use an array here (Object.keys)
and check if 0 = c, ...